src/assets/RestApi/rest-api.md

```diff
@@ -0,0 +1,448 @@
+---
+title: Understanding REST APIs - A Comprehensive Guide with Practical Example
+authorName: Prajwol Amatya
+authorAvatar: https://1.gravatar.com/avatar/de64e53c0e2cb393dd0d14ffdd53058ee9c607b35e366dd392425bd1b95a034c?size=256
+authorLink: https://github.com/prajwolamatya
+createdAt: May 8, 2024
+tags: rest, api, rest api, python, flask
+banner: https://blog.jankaritech.com/src/assets/RestApi/images/rest-api-banner.png
+---
+
+Within the field of web development, APIs (Application Programming Interface) are essential for facilitating communication across various software components. Because of its simplicity, scalability, and statelessness, REST (Representational State Transfer) APIs stand out among the other types of APIs. In this blog, we will dive into basics of building a REST API.
```

Comment:
```suggestion
Within the field of web development, APIs (Application Programming Interfaces) are essential for facilitating communication across various software components. Because of its simplicity, scalability, and statelessness, REST (Representational State Transfer) APIs stand out among the other types of APIs. In this blog, we will dive into basics of building a REST API.
```

I think you need plural here